translate/storage/flatxml.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+
+"""Module for handling flat XML files."""
+
+from lxml import etree
+
+from translate.lang import data
+from translate.misc.xml_helpers import getText, namespaced, reindent
+from translate.storage import base
+from translate.storage.placeables import general
+
+
+class FlatXMLUnit(base.TranslationUnit):
+    """A single term in the XML file."""
+
+    element_name = "str"
+    attribute_name = "key"
+    namespace = None
```

Comment:
I don't think the three class attributes above are necessary since they are already set per instance in `__init__`.

Reply:
Mostly copy&paste from the beginnings of this PR where I wasn't as familiar with how Python works. I suppose I can just drop them.